style.css

```diff
@@ -72,3 +68,145 @@ p {
   background-repeat: no-repeat;
   background-position: right top;
 }
+
+.section-1 {
+  margin-top: 40px;
+  background: #f7f7f9;
+  padding: 1rem;
+  border: 2px solid #e1e3e8;
+  height: auto;
+  width: 100%;
+}
+
+.first-card {
+  margin-top: 80px;
+  background: #fff;
+  padding: 1rem;
+  border: 2px solid #e1e3e8;
+  box-shadow: 0 10px 15px -3px rgba(0, 0, 0, 0.1);
+  border-radius: 1rem;
+  height: auto;
+  width: 100%;
+}
```

Comment:
- [ ] I have noticed that at width `375px` the contents of the card cross through the borders of the card. Kindly ensure that at atleast 375px width, your design matches the Figma template design 😎. In addition, this will help you to remove the horizontal scroll.

Your design at `375px`
![image](https://user-images.githubusercontent.com/99394374/228766651-3813811e-768d-4c74-92fc-dbe61c1eac68.png)

Figma design 😎
![image](https://user-images.githubusercontent.com/99394374/228766958-0456395a-b3c4-486f-861b-0036dcbbc0bb.png)